### PR TITLE
fix(auth-admin-web): Add urlencoding where missing.

### PR DESCRIPTION
**Pull request by valurefugl**
https://www.notion.so/Admin-Rest-endpoints-not-handling-8447297884d849e3a3476561b248034a
# ...

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/auth-admin-web/services/ClientService.ts [hunk 1]
```diff
@@ -213,7 +215,7 @@ export class ClientService extends BaseService {
   ): Promise<number | null> {
     return BaseService.DELETE(
       `client-grant-type/${encodeURIComponent(clientId)}/${encodeURIComponent(
-        grantType,
+        encodeURIComponent(grantType),
```
[the remark this answers is not in the document]
Comment (valurefugl) [reply apps/auth-admin-web/services/ClientService.ts]: Good catch, fixed.

File: apps/auth-admin-web/services/TranslationService.ts [hunk 2]
```diff
@@ -40,7 +40,7 @@ export class TranslationService extends BaseService {
 
   /** Gets language by it's isoKey */
   static async findLanguage(isoKey: string): Promise<Language | null> {
-    return BaseService.GET(`translation/language/${isoKey}`)
+    return BaseService.GET(`translation/language/${encodeURIComponent(isoKey)}`)
```
[the remark this answers is not in the document]
Comment (valurefugl) [reply apps/auth-admin-web/services/TranslationService.ts]: Removed.

File: apps/auth-admin-web/services/TranslationService.ts [hunk 3]
```diff
@@ -55,7 +55,9 @@ export class TranslationService extends BaseService {
 
   /** Deletes a Language */
   static async deleteLanguage(isoKey: string): Promise<number | null> {
-    return BaseService.DELETE(`translation/language/${isoKey}`)
+    return BaseService.DELETE(
+      `translation/language/${encodeURIComponent(isoKey)}`,
```
[the remark this answers is not in the document]
Comment (valurefugl) [reply apps/auth-admin-web/services/TranslationService.ts]: Removed.

**Review by saevarma — APPROVED** [review]
LGTM

**Review by unnarb — APPROVED** [review]

**Pull request merged**